catalog/.eslintrc.js

```diff
@@ -72,6 +69,7 @@ module.exports = {
     'react-hooks/rules-of-hooks': 'error',
     'redux-saga/no-yield-in-race': 2,
     'redux-saga/yield-effects': 2,
+    'react-hooks/exhaustive-deps': 2,
```

Comment:
@fiskus this is nice, but i think we should keep the A-Z order for consistency

Reply:
Got it. Will fix in the next PR